torch/csrc/jit/passes/onnx/eval_peephole.cpp

```diff
@@ -47,14 +47,17 @@ static void fuseConvBatchNorm(Block* b, ValueToParamPairMap& valsToParamsMap) {
       fuseConvBatchNorm(child_block, valsToParamsMap);
     }
     if (it->kind() == onnx::Conv) {
-      if (it->output()->uses().size() != 1) {
```

Comment:
We should need to check bn is the only user of this conv right?

Reply:
Just added this back. Thanks.